src/c/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(lgraph-cbindings STATIC c.cpp)
```

Comment:
The first thing to remember is that the declaration of **lgraph_api** is in `include` directory, and its definition is in `src/lgraph_api` directory.

Logically c-bindings is part of **lgraph_api**, now the problem is that C API has been added into `include`, but its definition is not packed in `${TARGET_LGRAPH}`. Please fix this in `BuildLGraphApi.cmake`.